src/connect/component.jsx

```diff
@@ -17,19 +17,35 @@ import { FPTI_KEY } from "@paypal/sdk-constants";
 
 const MIN_MAJOR_VERSION = 3;
 const MIN_MINOR_VERSION = 97;
-export const MIN_BT_VERSION = `${MIN_MAJOR_VERSION}.${MIN_MINOR_VERSION}.3-connect-alpha.6.1`; // Minimum for supporting AXO
+const MIN_PATCH_VERSION = 3;
+export const MIN_BT_VERSION = `${MIN_MAJOR_VERSION}.${MIN_MINOR_VERSION}.${MIN_PATCH_VERSION}-connect-alpha.6.1`; // Minimum for supporting AXO
+
+export const DEFAULT_BT_VERSION = `3.107.1`;
 
 export function getSdkVersion(version: string | null): string {
   if (!version) {
-    return MIN_BT_VERSION;
+    return DEFAULT_BT_VERSION;
   }
   const versionSplit = version.split(".");
+  // patch could have an alpha tag
+  const patchSplit = versionSplit[2].split("-");
```

Comment:
Me and Nik thought it something good to have but we can remove it.

Reply:
With the `DEFAULT_BT_VERSION` & `MIN_BT_VERSION` having patch versions greater that 0, it seemed like a valid check.

Reply:
the only thing i'm worried about is accidentally marking current integrations as invalid. Technically, if this check is working correctly, then the merchant's integration wouldn't work anyway.

but i would check with the fastlane team to see if they have any dashboards we can watch while the SDK deploys

Reply:
With the current logic anything below `3.97.3` will throw an error, the previous logic would only throw an error for versions less than `3.97`

It seems like the min version is on this specific version `3.97.3-connect-alpha.6.1`
If a merchant was on `3.97.1`, no error would be thrown, but I don't think that version would have the Fastlane changes? 

Is that logic correct?

Reply:
i don't think this will break anything, but i also don't think it's necessary.
just for context, the first official version of BT to support Fastlane is `3.104.0`. before that, the only other versions with Fastlane were:
`3.95.0-connect-alpha.12`
`3.97.3-connect-alpha.6.1`
`3.103.0-fastlane-beta.7.3`